.circleci/config.yml

```diff
@@ -155,6 +155,28 @@ commands:
       - android/save-gradle-cache
       - android/save-build-cache
 
+  install-ruby:
```

Comment:
Might be worth using the ruby orb for this: https://circleci.com/developer/orbs/orb/circleci/ruby#commands-install